workflow/schemas/config.schema.yaml

```
@@ -104,6 +104,46 @@ isoform_analysis:
+      minimum: 1
+  required:
+    - lambda
+  allOf:
+    - if:
+        properties:
+          lambda:
+            const: true
+      then:
+        required: ["uniref","num_matches"]
+
+protein_annotation:
```

YAMLlint (1.37.1)

[error] duplication of key "protein_annotation" in mapping

(key-duplicates)